infra/templates/ado-bootstrap-iac-pipeline/README.md

```diff
@@ -0,0 +1,193 @@
+# Bootstrap for Cobalt IaC Pipelines on Azure DevOps
+
+## Overview 
+This provides a starting point for anyone interested in having an Azure DevOps
+Project that will 'run' a target Cobalt template whenever that template changes.
+
+## Intended audience
+
+*FIXME:* Application developer that is brand new to Cobalt templating and it's *Cobalt Infrastructure Template* (CIT) developer workflow.
```

Comment:
Did you intend to keep the *FIXME* remark (and the others?)

Reply:
I forgot to add this in my review, but I think that the only content I see that is missing from this document is the following:

- A note/docs ref about setting up the ADO provider locally
- What to do after this template (fork cobalt, push to repo that was created, and then trigger a build)

Have you tested the latter to make sure it works?

Reply:
Tested!
![image](https://user-images.githubusercontent.com/4935046/80016400-b2ed1a00-8498-11ea-8b63-278f764a7121.png)